app/assets/stylesheets/application.scss

```diff
@@ -13,29 +13,46 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
- *= depend_on_asset "Landing_Page.jpg"
  */
 
  body {
-  background-image: url("<%= asset_path('Landing_Page.jpg') %>");
+  background-image: image_url("Landing_Page.jpg");
   background-size: cover;
   background-repeat: no-repeat;
   background-position: center top;
-  /*overflow-x: hidden;*/
+  background-position: 0px 45px;
```

Comment:
Property `background-position` already defined on line 22<br>`0px` should be written without units as `0`